packages/tupaia-web-server/src/utils/generateFrontendExcludedFilter.ts

```diff
@@ -8,17 +8,25 @@ interface FrontEndExcludedConfig {
 }
 
 // In the db project.config.frontendExcluded is an array with one entry for some reason
-export function generateFrontendExcludedFilter({
-  frontendExcluded,
-}: {
-  frontendExcluded: FrontEndExcludedConfig[] | undefined;
-}) {
+export function generateFrontendExcludedFilter(
+  {
+    frontendExcluded,
+  }: {
+    frontendExcluded: FrontEndExcludedConfig[] | undefined;
+  },
+  defaultFrontendExcludedFields: string[] = [],
+) {
   return frontendExcluded
     ? {
         type: {
           comparator: '!=',
           comparisonValue: frontendExcluded[0].types,
         },
       }
-    : {};
+    : {
+        type: {
+          comparator: '!=',
+          comparisonValue: defaultFrontendExcludedFields,
+        },
+      };
```

Comment:
Should we default here? Or combine?

Reply:
I think we default, seems like the `web-config-server` was either doing one or the other